units/latest.pxu

```diff
@@ -1,104 +1,6 @@
-id: edgex/latest/install
-_summary: Test installation of EdgeX snap
-command: $PLAINBOX_PROVIDER_DATA/latest/test-install.sh
+id: edgex/latest/run-go-tests
+_summary: Test edgexfoundry snap with Go
+command: $PLAINBOX_PROVIDER_DATA/latest/run-go-tests.sh
```

Comment:
```suggestion
id: edgex/latest/edgexfoundry
_summary: Test edgexfoundry snap with Go
command: $PLAINBOX_PROVIDER_DATA/latest/run-go-tests.sh edgexfoundry

id: edgex/latest/edgexfoundry/refresh
_summary: Test edgexfoundry snap with Go
command: $PLAINBOX_PROVIDER_DATA/latest/run-go-tests.sh edgexfoundry/refresh

id: edgex/latest/edgex-no-sec
_summary: Test edgexfoundry snap with Go
command: $PLAINBOX_PROVIDER_DATA/latest/run-go-tests.sh edgex-no-sec
```

Reply:
The refresh tests should stay disabled (commended out) for now until we release v3 and wanna test upgrades from v3 to higher.

The edgex-no-sec tests need the SERVICE_CHANNEL which is used for edgex-ekuiper and edgex-device-virtual installs. Hardcoding it to `latest/beta` sounds good but we don't use that channel on those snaps right now; it falls back to latest/stable. Maybe we can leave out this test too (commented out) for now until we have a better plan.